pkg/operator/certrotation/signer.go

```diff
@@ -58,6 +58,10 @@ type RotatedSigningCASecret struct {
 // EnsureSigningCertKeyPair manages the entire lifecycle of a signer cert as a secret, from creation to continued rotation.
 // It always returns the currently used CA pair, a bool indicating whether it was created/updated within this function call and an error.
 func (c RotatedSigningCASecret) EnsureSigningCertKeyPair(ctx context.Context) (*crypto.CA, bool, error) {
+	return c.ensureSigningCertKeyPair(ctx, time.Now())
```

Comment:
I'd prefer changing `RotatedSigningCASecret` to accept `clock.Clock` and then perhaps this method could provide a default implementation if not set.